background/rust.tex

```diff
@@ -455,7 +546,7 @@ \subsection{Cargo}
 \subsubsection{Project Structure}
 
 Every crate, or package, built by \texttt{Cargo} requires a \texttt{Cargo.toml} file to be present in the root directory.
-This file is interpreted by \texttt{Cargo} and is used to determine the name of library and executables to be built in the package and what other packages that it depends on.
+This file is interpreted by \texttt{Cargo} and is used to determine the name of the library and executables to be built in the package and what other packages that it depends on.
 It also includes information that tells \texttt{Cargo} how the package can be compiled for different target architectures, and it is used to define different \textit{features} of a package - a way to conditionally compile certain parts of the code present in the library.
 
 \begin{listing}
```

Comment:
Didn't we agree to simplify this structure?

I suggest something like this as the first example:

```
\dirtree{%
.1 hello\_world.
.2 Cargo.lock.
.2 src.
.3 main.rs.
}
```